api/pokemon.go

```diff
@@ -0,0 +1,24 @@
+package api
+
+import (
+	"net/http"
+
+	"github.com/gin-gonic/gin"
```

Comment:
please sort the imports

Reply:
I used the default sorting format of Golang already

Reply:
Yes by default it will just sort it this way, if you correctly sort this whenever a new package is imported it will automatically sort them correctly.